package.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name"            : "eslint-config-walmart",
-  "description"     : "A set of default eslint configurations, Walmart Labs style.",
+  "name"            : "eslint-config-formidable",
+  "description"     : "A set of default eslint configurations",
 
   "version"         : "1.0.0",
   "author"          : "Eric Baer <me@ericbaer.com>",
```

Comment:
Update author to "authors" (by including yourself) and use everyone's `@formidable.com` address instead of personal one...